src/components/DataTypes/index.ts

```diff
@@ -25,7 +25,7 @@ export const getDataModel = (schema?: JSONSchema) => {
 		return arrayType;
 	}
 	if (type === 'string') {
-		if (format === 'date-time') {
+		if (format === 'date-time' || format === 'elapsed-time') {
```

Comment:
Tempted to also add an `includes('date-time')` to also allow custom extraFormats